qutebrowser/completion/completionwidget.py

```diff
@@ -386,8 +386,12 @@ def completion_item_yank(self, sel=False):
         Args:
             sel: Use the primary selection instead of the clipboard.
         """
-        index = self.currentIndex()
-        if not index.isValid():
-            raise cmdexc.CommandError("No item selected!")
-        data = self.model().data(index)
-        utils.set_clipboard(data, selection=sel)
+        status = objreg.get('status-command', scope='window',
+                            window=self._win_id)
+        text = status.selectedText()
+        if not text:
+            index = self.currentIndex()
+            if not index.isValid():
+                raise cmdexc.CommandError("No item selected!")
```

Comment:
What do you think about yanking `status.text()` here instead? It'd make it easy to try something, recall it from history, and then copy it to e.g. show it someone in IRC.

Reply:
I thought about that. For me the compelling use case is copying a url to show someone, where I wouldn't want the leading `:open`

Reply:
I might take a look at #2453. If I can open the current command in my editor, I can easily yank whatever part of it I want, plus it is a useful feature in its own right.

Reply:
Fair enough - it's probably hard to define what ctrl-c without a selection would do.